.rubocop.yml

```diff
@@ -7,8 +7,43 @@ AllCops:
   - ./node_modules/**/*
   - ./vendor/**/*
 
-Metrics/LineLength:
+Layout/LineLength:
   Max: 100
 
 Style/Documentation:
   Enabled: false
+
```

Comment:
Not a big deal, but would it be beneficial to alphabetize-merge with the ones that were there before (like Layout/Linelength and Style/Documentation)